dev/com.ibm.ws.webcontainer.servlet.3.1/src/com/ibm/ws/webcontainer31/srt/SRTServletRequest31.java

```diff
@@ -63,14 +63,15 @@ public class SRTServletRequest31 extends SRTServletRequest implements HttpServle
     
     public SRTServletRequest31(SRTConnectionContext31 context)
     {
-        this._connContext = context;
-        this._requestContext = new SRTRequestContext31(this);
-        this._in = createInputStream();
-            if (TraceComponent.isAnyTracingEnabled()&&logger.isLoggable (Level.FINE)) {  //306998.15
-                logger.logp(Level.FINE, CLASS_NAME,"SRTServletRequest", "this->"+this+": " + "inputStream is of type --> " + this._in);
-        }
+        super(context);
+        this._requestContext = createRequestContext();
```

Comment:
```suggestion
```

The whole point was not to do it here, but to have it be done in the parent class.